utils/release/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@coveo/release",
+  "private": true,
+  "authors": "Coveo",
+  "version": "1.0.0",
+  "type": "module",
+  "dependencies": {
+    "@coveo/semantic-monorepo-tools": "1.5.22",
+    "async-retry": "1.3.3",
+    "conventional-changelog-angular": "5.0.13",
+    "npm-registry-fetch": "14.0.3",
+    "octokit": "2.0.14",
+    "semver": "7.3.8",
+    "ts-dedent": "2.2.0"
+  },
+  "bin": {
+    "coveo-release-1": "./phase1-bump-all-packages.mjs",
+    "coveo-release-2": "./phase2-git-commit-tag-push.mjs"
```

Comment:
I think both of these scripts should be renamed to something other than `1` and `2`.

`coveo-release-1` could instead be named something like `auto-bump-version`. Its script file could also be renamed to remove "all" from its name.

`coveo-release-2` could be named something like `commit-and-push-bumps`.

Reply:
Script renamed!